api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/Pagination.scala

```diff
@@ -13,8 +18,12 @@ trait Paginated { this: QueryParams =>
         .filterNot(_ >= 1)
         .map(_ => "page: must be greater than 1"),
       pageSize
-        .filterNot(size => size >= 1 && size <= 100)
-        .map(_ => "pageSize: must be between 1 and 100")
+        .filterNot { size =>
```

Comment:
Fun fact: I learned the other day about how Scala treats contiguous ranges as special cases and can do certain operations in O(1) upon them, so you could do (not that I'm necessarily recommending it)

```scala
pageSize
  .filterNot { size =>
	import PaginationLimits._
    minSize to maxSize contains size
  }
```

Reply:
Nice! But yes, I think I will not do that today.